### PR TITLE
fix(config): update to use oauth server in auth directory

### PR DESCRIPTION
**Pull request by vbudhram**
Connects to https://github.com/mozilla/fxa-auth-server/issues/2679

@mozilla/fxa-devs r?

### CODE REVIEW

**Comment by rfk** [comment]
> [PM2][ERROR] script not found : /home/travis/build/mozilla/fxa-local-dev/fxa-oauth-server/bin/internal.js

CI failing about ^

**Comment by vbudhram** [comment]
@rfk Oh no..I just forced pushed a fix and overwrote your commit. I think we had the same fix lol, lets see what CI says.

**Comment by rfk** [comment]
😆

File: _scripts/oauth_mysql.sh [hunk 1]
```diff
@@ -5,5 +5,5 @@ check_mysql
 mysqlStarted=$?
 
 if [ "$mysqlStarted" ]; then
-  node ../fxa-oauth-server/bin/server.js 
+  node ../fxa-auth-server/fxa-oauth-server/bin/server.js
```
Comment (rfk) [inline _scripts/oauth_mysql.sh]: Due to updating `"cwd": "fxa-auth-server/fxa-oauth-server",` you are now running this within the `fxa-auth-server/fxa-oauth-server` directory, so I think you need to replace `../` with `../../` in two places in this file.

Comment (rfk) [reply _scripts/oauth_mysql.sh]: (not going to attempt another push-my-own-fix on your PR though 😆)

**Pull request merged**